components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -2460,6 +2460,33 @@ public Group createGroup(Group group, Map<String, Boolean> requiredAttributes)
         return group;
     }
 
+    private void validateUserstoreDomain(String domainName) throws CharonException {
+
+        if (isInternalOrApplicationGroup(domainName)) {
+            return;
```

Comment:
shall we add a debug log here

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/394/commits/0d947206fe849a0894717972aaf562522a025056